services/minespace-web/common/constants/strings.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/jsx-key */
```

Comment:
Was getting "Missing key prop for element in array" on this file for MAP_UNAVAILABLE and CONTACT_ADMIN, which just didn't seem to make any sense in this context (lines 26, 35).